assignments/objects.js

```diff
@@ -10,38 +10,81 @@
 // 5,adaine5@samsung.com,Antonietta,F
 
 // Example format of an intern object: 1,examples@you.edu,Example,F
-const example = {
-  "id": 0,
-  "name": "Example",
-  "email": "examples@you.edu",
-  "gender": "F"
-}
+
+// const example = {
+//   "id": 0,
+//   "name": "Example",
+//   "email": "examples@you.edu",
+//   "gender": "F"
+// }
 
 // Write your intern objects here:
 
+const internOne = {
+  "id": 1,
+  "name": "Mitzi",
+  "email": "mmelloy0@psu.edu",
+  "gender": "F"
+}
+const internTwo = {
+  "id": 2,
+  "name": "Kennan",
+  "email": "kdiben1@tinypic.com",
+  "gender": "M"
+}
+const internThree = {
+  "id": 3,
+  "name": "Kevin",
+  "email": "kmummery2@wikimedia.org",
+  "gender": "M"
+}
+const internFour = {
+  "id": 4,
+  "name": "Gannie",
+  "email": "gmartinson3@illinois.edu",
+  "gender": "M"
+}
+const internFive = {
+  "id": 5,
+  "name": "Antonietta",
+  "email": "adaine5@samsung.com",
+  "gender": "F"
+}
 
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
+console.log(internOne.name)
 
 // Kennan's ID
+console.log(internTwo.id)
 
 // Keven's email
+console.log(internThree.email)
 
 // Gannie's name
+console.log(internFour.name)
 
 // Antonietta's Gender
+console.log(internFive.gender)
 
 // ==== Challenge 3: Object Methods ==== 
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
 // console.log(kennan.speak());
+console.log("Hello, my name is " + internTwo.name + "!")
```

Comment:
I think the assignment wanted you to add a method to the object, and then invoke the method.  but this also solves the problem, so can't blame ya!